ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/node-type-l3-interfaces-bgp.yml

```diff
@@ -0,0 +1,50 @@
+---
+type: wan_router
+# By default it means that default routing protocol is none
+# we make it ebgp to trigger also when uplink-type is lan
+underlay_routing_protocol: ebgp
```

Comment:
do we want this?

Reply:
yeah the goal was to make sure that underlay router is False not because of underlay_routing_protocol but because of uplink_type